Userland/Libraries/LibJS/Bytecode/Pass/UnifySameBlocks.cpp

```diff
@@ -24,6 +24,8 @@ void UnifySameBlocks::perform(PassPipelineExecutable& executable)
         auto& block = executable.executable.basic_blocks[i];
         auto block_bytes = block.instruction_stream();
         for (auto& candidate_block : executable.executable.basic_blocks.span().slice(i + 1)) {
+            if (equal_blocks.contains(&*candidate_block))
```

Comment:
This is a hash map though, at best it would overwrite the old entry :thinking:

Reply:
Well we do a full `memcmp` over both blocks, so I guessed that this fast path is a cheaper alternative